editor/source/components/HexMetrics.js

```diff
@@ -89,20 +89,22 @@ export default class HexCount extends React.Component {
     const rows = metrics.map((count) => {
       const metric = isNaN(count.metric) ? <td /> : <td>{count.metric}</td>
       const ratio = isNaN(count.ratio) ? <td /> : <td>{count.ratio}</td>
-      const deviation = isNaN(count.deviation) ? <td /> : <td>{count.deviation}</td>
+      const adjustString = isNaN(count.deviation) ? '' : count.deviation > 0 ? `+${count.deviation}` : count.deviation
+      const adjust = <td>{adjustString}</td>
       return (
-        <tr key={count.key}>
+        <tr
+          key={count.key}
+          id={count.key}
+          onMouseOver={this.props.onMetricMouseOver}
```

Comment:
It might be nice to do `{event => this.props.onMetricMouseOver(event.currentTarget.id)}` here instead of passing the whole `event` back to `GridGraphic`. What do you think?